Sakura/database/users.py

```
@@ -0,0 +1,87 @@
+                    ON CONFLICT (user_id)
+                    DO UPDATE SET
+                        username = EXCLUDED.username,
+                        first_name = EXCLUDED.first_name,
+                        last_name = EXCLUDED.last_name,
+                        updated_at = CURRENT_TIMESTAMP
+                """, user_id, username, first_name, last_name)
+            logger.debug(f"💾 User {user_id} saved to database")
+        except Exception as e:
+            logger.error(f"❌ Failed to save user {user_id} to database: {e}")
+
+    asyncio.create_task(save_user())
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)